### PR TITLE
Enhanced fiotest.py for auto mount and run without passing disk

### PR DESCRIPTION
**Pull request by Naresh-ibm**
*Added auto mount/unmount using partition utility
*made to run even without disk passed
*made to pass the fio-tool url from yaml file

Signedh-off-by: Naresh Bannoth <nbannoth@in.ibm.com>

### CODE REVIEW

**Comment by Naresh-ibm** [comment]
[fiotest-without-disk.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/830814/fiotest-without-disk.txt)
[Uploading fiotest-with-disk.txt…]()

**Comment by PraveenPenguin** [comment]
please use Signed-off-by by `git` using -`s` and reabse it make Travis happy

File: io/disk/fiotest.py [hunk 1]
```diff
@@ -44,21 +45,46 @@ def setUp(self):
         """
         Build 'fio'.
         """
-        tarball = self.fetch_asset('http://brick.kernel.dk/snaps/fio-2.1.10.tar.gz')
+        self.url = self.params.get('fio_tool_url', default=None)
+        # http://brick.kernel.dk/snaps/fio-2.1.10.tar.gz
+        tarball = self.fetch_asset(self.url)
```
Comment (narasimhan-v) [inline io/disk/fiotest.py]: What happens here if `self.url` is `None` ?

**Comment by Naresh-ibm** [comment]
closing this PR in favour of #320

**Pull request closed without merging**